guides/v2.1/cloud/reference/docker-config.md

```diff
@@ -80,27 +88,69 @@ The `ece-tools` package provides a `docker:build` command to generate the Docker
     ```bash
     docker-compose run build cloud-build
     ```
-    
+
     * Deploy Magento in the Docker container:
-    
+
     ```bash
     docker-compose run deploy cloud-deploy
     ```
 
-1. Access your local Magento Cloud template by opening one of the following secure URLs in a browser:
- 
-    -  [`http://localhost:8080`](http://localhost:8080){:target="\_blank"}
-    -  [`https://localhost`](https://localhost){:target="\_blank"}
+1.  Open the `http://localhost:8080` URL in a browser to access your local Magento Cloud template.
+
 
-#### To stop and remove the Cloud Docker environment:
+#### To launch Docker with `ece-tools` v2002.0.12 and later:
 
-Remove all components of your local Magento Cloud template including containers, networks, volumes, and images.
+1.  Download a template from the [Magento Cloud repository](https://github.com/magento/magento-cloud){:target="\_blank"}.
+
+1.  Add your credentials to `auth.json` file.
+
+1.  Update the template dependencies.
+
+    ```bash
+    composer install
+    ```
+
+1.  In your local environment, start the Docker configuration generator.
+
+    ```bash
+    vendor/bin/ece-tools docker:build
+    ```
+
+1.  Copy the configuration files.
+
+    ```bash
+	cp docker/config.env.dist docker/config.env
+    ```
+
+    ```bash
+	cp docker/global.env.dist docker/global.env
+    ```
+
+1.  Build files to containers and run in the background.
+
+    ```bash
+    docker-compose up -d --build
+    ```
+
+1.  Install Magento. This step may take some time to complete.
+
+    ```bash
+    docker-compose run cli magento-installer
+    ```
+
+1.  Open the `http://localhost:8080` URL in a browser to access your local Magento Cloud template.
+
+
+## Stop and remove the Docker configuration
+
+Remove all components of your local Magento Cloud Docker instance including containers, networks, volumes, and images.
 
 ```bash
 docker-compose down
 ```
 
-### Integration testing with ece-tools
+## Use the `ece-tools` package for integration testing
```

Comment:
try to avoid using ece-tools package in the heading. Also, if this branch is up to date, then the variable for ece-tools is available.